src/range/padding.rs

```diff
@@ -40,12 +39,14 @@ impl Serializable for RangeProofPadding {
     fn serialize(&self) -> Vec<u8> {
         let mut result: Vec<u8> = Vec::new();
         let mut bytes = self.get_aggregated().to_bytes();
-        result.append(&mut usize_to_bytes(bytes.len(), PROOF_SIZE_BYTE_NUM));
```

Comment:
Removing `PROOF_SIZE_BYTE_NUM` check here is fine because it checks that the result is < 2^64 bits which is the limit of the usize type anyway, so the input could never be greater than that...